tmt/steps/provision/testcloud.py

```diff
@@ -575,8 +575,11 @@ def remove(self):
                     f"Failed to remove testcloud instance: {error}")
             self.info('guest', 'removed', 'green')
 
-    def reboot(self, hard=False):
+    def reboot(self, hard=False, command=None):
         """ Reboot the guest, return True if successful """
+        # Use custom reboot command if provided
```

Comment:
I also considered a similar solution to what you ended up using for the reboot. My reasoning behind going with the former approach was that if I was now to add a new provision plugin, there is one more thing to keep in mind - handling of `command` in `reboot`. But when I see the code now, it does not look as bad as I thought it would, so lets go with your solution :+1: